bigtable-client-core-parent/bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableAdmin.java

```diff
@@ -100,7 +98,7 @@
   private final BigtableHBaseSettings settings;
   protected final CommonConnection connection;
   protected final AdminClientWrapper adminClientWrapper;
-  private BigtableClusterName bigtableSnapshotClusterName;
+  private ClusterName bigtableSnapshotClusterName;
```

Comment:
I've been wanting to drop the gapic generated resource names. Can we stick to a String here?